data/bash-completion/fwupdtool.in

```diff
@@ -9,6 +9,7 @@ _fwupdtool_cmd_list=(
 	'firmware-export'
 	'firmware-extract'
 	'firmware-parse'
+	'firmware-sign'
```

Comment:
I don't recall for sure but does this also need syncing to fish completion?

Reply:
fish only has completion logic for fwupdmgr unfortunately.